tests/test_wagon.py

```diff
@@ -66,10 +62,6 @@ def test_run(self):
         proc = wagon._run('uname')
         assert proc.returncode == 0
 
-    def test_run_bad_command(self):
```

Comment:
without `shell=True`, this will just return a IOError (no such file or directory) instead.
I'm not really willing (or at least for now) to spend time adding tests for that since, well, we only call known stuff, and `_run` is not exposed to the user